### ISSUE TITLE
Tracking CI testing downstream

### ISSUE THREAD

**Issue by jaimergp** [conda-forge/openmm-feedstock#37]
As we move forward with the migration (almost done!), I'd like some more testing in real world scenarios. Packages for Linux, Windows and OSX are now available under [`conda-forge/label/test`](https://anaconda.org/conda-forge/openmm/files?channel=test), together with the CF-built `cudatoolkit` packages. Take into account that these are built off a commit very close to the future v7.5.0.

I have enabled branches for the CI of the following projects, and I am comparing results against the `omnia` and `omnia-dev` packages. Results are positive:

- 🟡 `choderalab/openmmtools`: one error not related to packaging since it's also happening in the omnia-dev nightlies.
- 🟢 `choderalab/perses`: all good.
- 🟡 `choderalab/yank`: errors unrelated to packaging.
- 🟢 `openforcefield/openforcefield`: all good.

If you are using `omnia[-dev]::openmm` in your CI, I'd suggest adding a new branch before the release. Check [`perses` CI implementation](https://github.com/choderalab/perses/blob/master/.github/workflows/CI.yaml#L54-L76) for some inspiration if needed.

***

If you want to test locally, something like this should work:

```
# auto select CUDA if available on your system
conda create -n openmm-cf -c conda-forge/label/test -c conda-forge openmm
# or specify CUDA manually
conda create -n openmm-cf -c conda-forge/label/test -c conda-forge openmm cudatoolkit=10.2

# activate
conda activate openmm-cfg

# basic tests
python -m simtk.testInstallation
```

**Comment by mattwthompson** [comment]
Giving it a college try here: https://github.com/openforcefield/openforcefield/pull/769

**Comment by jaimergp** [comment]
I ran a the benchmark scripts on my Windows machine with a RTX2070 eGPU to check everything runs smoothly. These are the numbers:

```
% python .\benchmark.py --platform CUDA --seconds 30 --precision mixed
Platform: CUDA
Precision: mixed

Test: gbsa
Step Size: 4 fs
Integrated 52923 steps in 29.9631 seconds
610.423 ns/day

Test: rf
Step Size: 4 fs
Integrated 22273 steps in 25.5346 seconds
301.455 ns/day

Test: pme (cutoff=0.9)
Step Size: 4 fs
Integrated 20771 steps in 26.2779 seconds
273.175 ns/day

Test: apoa1rf
Step Size: 4 fs
Integrated 13613 steps in 30.483 seconds
154.337 ns/day

Test: apoa1pme
Step Size: 4 fs
Integrated 8272 steps in 28.1456 seconds
101.572 ns/day

Test: apoa1ljpme
Step Size: 4 fs
Integrated 7072 steps in 30.2683 seconds
80.7473 ns/day

Test: amoebagk (epsilon=1e-05)
Step Size: 2 fs
Integrated 686 steps in 27.3976 seconds
4.32669 ns/day

Test: amoebapme (epsilon=1e-05)
Step Size: 2 fs
Integrated 1829 steps in 29.2349 seconds
10.8107 ns/day

% python .\benchmark.py --platform OpenCL --seconds 30 --precision mixed
Platform: OpenCL
Precision: mixed

Test: gbsa
Step Size: 4 fs
Integrated 32422 steps in 28.2458 seconds
396.698 ns/day

Test: rf
Step Size: 4 fs
Integrated 33278 steps in 29.3326 seconds
392.085 ns/day

Test: pme (cutoff=0.9)
Step Size: 4 fs
Integrated 25424 steps in 28.6362 seconds
306.833 ns/day

Test: apoa1rf
Step Size: 4 fs
Integrated 12545 steps in 29.6115 seconds
146.415 ns/day

Test: apoa1pme
Step Size: 4 fs
Integrated 9192 steps in 30.1728 seconds
105.285 ns/day

Test: apoa1ljpme
Step Size: 4 fs
Integrated 7372 steps in 30.3474 seconds
83.9533 ns/day

Test: amoebagk (epsilon=1e-05)
Step Size: 2 fs
Traceback (most recent call last):
  File ".\benchmark.py", line 156, in <module>
    runOneTest(test, args)
  File ".\benchmark.py", line 112, in runOneTest
    context = mm.Context(system, integ, platform, properties)
  File "C:\Users\Jaime\Miniconda3\envs\openmm-cf-cu\lib\site-packages\simtk\openmm\openmm.py", line 5778, in __init__
    _openmm.Context_swiginit(self, _openmm.new_Context(*args))
simtk.openmm.OpenMMException: Specified a Platform for a Context which does not support all required kernels

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".\benchmark.py", line 158, in <module>
    print('Test failed: %s' % ex.message)
AttributeError: 'OpenMMException' object has no attribute 'message'
```

@peastman -- are those `amoebagk` errors on OpenCL expected?

**Comment by jaimergp** [comment]
Closed by #41
